### PR TITLE
fix: python generate unusable class for empty properties

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
If the class generator tries to render one with no properties, the class file is unable to compile. 

This change is part of fixing the black-box tests.

### CODE REVIEW

File: src/generators/python/renderers/ClassRenderer.ts [hunk 1]
```diff
@@ -76,13 +76,20 @@ export const PYTHON_DEFAULT_CLASS_PRESET: ClassPresetType<PythonOptions> = {
...
+        if (!property.required) {
+          return `if hasattr(input, '${property.propertyName}'):\n\tself._${property.propertyName} = input.${property.propertyName}`;
+        }
+        return `self._${property.propertyName} = input.${property.propertyName}`;
+      });
+      body = renderer.renderBlock(assigments);
+    } else {
+      body = `"""
+No properties
+"""`;
```
Comment (magicmatatjahu) [inline src/generators/python/renderers/ClassRenderer.ts:91]: Do we need that comment?

**Review by magicmatatjahu — APPROVED** [review]

File: src/generators/python/renderers/ClassRenderer.ts (hunk 1 above)
Comment (jonaslagoni) [reply src/generators/python/renderers/ClassRenderer.ts:91]: Not necessarily that comment, but yes, otherwise compilation fails 😅

Comment (jonaslagoni) [reply src/generators/python/renderers/ClassRenderer.ts:91]: Maybe `pass` is better? 🤔 https://www.programiz.com/python-programming/pass-statement

Comment (magicmatatjahu) [reply src/generators/python/renderers/ClassRenderer.ts:91]: No, comment is good :) I don't have a problem with it, I only asked.

**Pull request merged**